dom/events/Event-defaultPrevented-after-dispatch.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
```

Comment:
Is there any particular reason to add this to files that contain only ASCII characters?